quarkchain/cluster/simple_network.py

```diff
@@ -25,6 +25,10 @@
 from quarkchain.utils import Logger
 
 
+ROOT_BLOCK_BATCH_SIZE = 100
+ROOT_BLOCK_HEADER_LIST_LIMIT = 500
```

Comment:
Could we find any reference number from Ethereum?  Looks like go-ethereum uses 128 and 512 respectively.

https://github.com/ethereum/go-ethereum/blob/master/eth/downloader/downloader.go